### PR TITLE
AAE-8296 springdoc migration changes - fix acceptance tests

### PR DESCRIPTION
**Pull request by jsokolowskii**

### CODE REVIEW

File: alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/patch.sh [hunk 1]
```diff
@@ -53,10 +53,17 @@ find ${GENERATED_SOURCE_DIR} -type f -iname 'ProjectsApi.java' -exec sed \
   -i $EDIT_FILE_IN_PLACE_PARAM {} +
 
 find ${GENERATED_SOURCE_DIR} -type f -iname 'ModelsApi.java' -exec sed \
-  -e 's/ResponseEntity<Void> getModelContent/byte[] getModelContent/' \
+  -e 's/ResponseEntity<Void> getModelDiagram1/byte[] getModelContent/' \
```
Comment (jesty) [inline alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/patch.sh:56]: do we have other methods with an incremental number?

**Review by jesty — COMMENTED** [review]
just a question

File: alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/patch.sh (hunk 1 above)
Comment (jsokolowskii) [reply alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib/patch.sh:56]: Yes, there are other methods (not with this name) that also have numbers.

**Review by jesty — APPROVED** [review]
LGTM

**Review by jesty — CHANGES_REQUESTED** [review]
I cannot change the message, don't do it, for now, (please change the version from 5.1.5-SNAPSHOT to 5.2.0-SNAPSHOT)

**Review by jesty — APPROVED** [review]

**Pull request merged**